react-backend/client/src/components/admin/adminUsersPage.js

```diff
@@ -146,18 +164,32 @@ export default class AdminUsersPage extends Component {
 
   getAgeGroupsForChart = (ageGroups) => {
     if (!ageGroups) return [];
-    return ageGroups.map(({ agegroups, agecount }, index) => ({
-      value: agecount,
-      color: this.getColorPair(index).color,
-      highlight: this.getColorPair(index).highlight,
-      label: agegroups,
-    }));
+
+    const ageData = {
```

Comment:
🌈 Colour me bruv...

Reply:
I think I'm gonna remove colours entirely for now, as its quite an important thing I feel we need to discuss:
- What colours?
- Should colours for gender and age group graphs be the same every time?
- There can be an infinite amount of activities so should the colours be randomly generated?
- If we do randomly generate colours, should we make sure that each colour is different enough from the others to be easily recognisable?
- Should the colours for gender be the standard blue/red?
- If we don't use the blue/red standard should we actively avoid these colours in the gender graph, to minimise confusion?